src/App.jsx

```diff
@@ -1,28 +1,97 @@
 import React from 'react';
 import './App.scss';
 import './styles/general.scss';
+import { getTodos } from './api/api';
 import { TodoList } from './components/TodoList';
 import { CurrentUser } from './components/CurrentUser';
 
 class App extends React.Component {
   state = {
     todos: [],
     selectedUserId: 0,
+    statusFilter: 'All',
+    titleFilter: '',
   };
 
+  componentDidMount = async() => {
+    const todos = await getTodos();
+
+    this.setState({
+      todos: todos.data.filter(todo => todo.title && todo.id && todo.userId),
+    });
+  }
+
+  handleSelect = (event) => {
+    this.setState({
+      statusFilter: event.target.value,
+    });
+  }
+
+  handleChange = (input) => {
```

Comment:
```suggestion
  handleInputChange = (input) => {
```
краще якось конкретизувати назву методу

Reply:
Дякую. Змінив